README.md

```diff
@@ -573,6 +573,13 @@ Verify flags:
 - :nerd_face: `--verify`: Verify the image (none|cosign). See [`docs/cosign.md`](./docs/cosign.md) for details.
 - :nerd_face: `--cosign-key`: Path to the public key file, KMS, URI or Kubernetes Secret for `--verify=cosign`
 
+Fanotify flags:
+- :nerd_face: `--fanotify`: Enable fanotify to record accessed files when task running.
+- :nerd_face: `--wait-time`: Available when set `--fanotify` flag. The run time of the `fanotify server` to record accessed files.
+- :nerd_face: `--wait-line`: Available when set `--fanotify` flag. The fanotify server runs until it detaches the string line from the container standard output.
+- :nerd_face: `--wait-signal`: Available when set `--fanotify` flag. The fanotify server runs until it gets the SIGINT (ctrl + C) signal.
+- :nerd_face: `--output`: The output path for fanotify.
+
```

Comment:
The flag set seem really confusing to me.
So I suggest using external OCI hooks as I commented above, but if this is not implementable as an OCI hook, at least the flag should be a CSV like `--record-file-access driver=fanotify,wait-time=30s,wait-line="foo",wait-signal=SIGINT,output=/path/to/out`, and has to be marked as experimental.

Reply:
Yes, the flag should be replaced with a CSV like format. I will try it with OCI Hook first.

Reply:
To make it with hook, you may rely on the `internal` subcommand to trigger the `fanotify` workflow, we already implemented some hooks [here]( https://github.com/containerd/nerdctl/blob/main/cmd/nerdctl/run.go#L938)